Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM quay.io/ncigdc/base-u18
+
+# work around issue with maven package in slim docker
+RUN mkdir -p /usr/share/man/man1 && mkdir /usr/lib/jvm
+
+RUN apt-get update && apt-get install -y wget gnupg git nano \
+    --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+RUN wget --no-check-certificate https://archive.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz && \
+    wget --no-check-certificate https://nexus.osdc.io/repository/misc/oracle/jdk/jdk-8u301-linux-x64.tar.gz && \
+    wget --no-check-certificat https://fastdl.mongodb.org/tools/db/mongodb-database-tools-ubuntu1804-x86_64-100.5.0.tgz
+
+RUN tar -xzf apache-maven-3.2.5-bin.tar.gz --directory /usr/local && \
+    tar -xvf mongodb-database-tools-ubuntu1804-x86_64-100.5.0.tgz --directory /usr/local && \
+    tar -xvf jdk-8u301-linux-x64.tar.gz --directory /usr/lib/jvm && \
+    rm -rf jdk-8u301-linux-x64.tar.gz && rm -rf apache-maven-3.2.5-bin.tar.gz && rm mongodb-database-tools-ubuntu1804-x86_64-100.5.0.tgz && \
+    rm -rf /usr/lib/jvm/*src.zip \
+           /usr/lib/jvm/lib/missioncontrol \
+           /usr/lib/jvm/lib/visualvm \
+           /usr/lib/jvm/lib/*javafx* \
+           /usr/lib/jvm/jre/lib/plugin.jar \
+           /usr/lib/jvm/jre/lib/ext/jfxrt.jar \
+           /usr/lib/jvm/jre/bin/javaws \
+           /usr/lib/jvm/jre/lib/javaws.jar \
+           /usr/lib/jvm/jre/lib/desktop \
+           /usr/lib/jvm/jre/plugin \
+           /usr/lib/jvm/jre/lib/deploy* \
+           /usr/lib/jvm/jre/lib/*javafx* \
+           /usr/lib/jvm/jre/lib/*jfx* \
+           /usr/lib/jvm/jre/lib/amd64/libdecora_sse.so \
+           /usr/lib/jvm/jre/lib/amd64/libprism_*.so \
+           /usr/lib/jvm/jre/lib/amd64/libfxplugins.so \
+           /usr/lib/jvm/jre/lib/amd64/libglass.so \
+           /usr/lib/jvm/jre/lib/amd64/libgstreamer-lite.so \
+           /usr/lib/jvm/jre/lib/amd64/libjavafx*.so \
+           /usr/lib/jvm/jre/lib/amd64/libjfx*.so
+
+COPY . /dcc-import
+# Set environment
+ENV JAVA_HOME /usr/lib/jvm/jdk1.8.0_301
+ENV PATH ${PATH}:${JAVA_HOME}/bin
+ENV PATH ${PATH}:/usr/local/apache-maven-3.2.5/bin
+ENV PATH ${PATH}:/usr/local/mongodb-database-tools-ubuntu1804-x86_64-100.5.0/bin
+
+WORKDIR /dcc-import
+
+# Install dependencies
+RUN mvn dependency:copy-dependencies
```

Comment:
Do we really need this step? This is useful when the application has an executable that can be executed outside of maven using something like `java --jar xxx.jar`. Since this is not the case for this project, we can save some docker image build time by skipping this

Reply:
If I recall, when I used mvn install it still had to download artifacts from OICR's Artifactory